server/utils/applicationUtils.ts

```diff
@@ -44,4 +49,41 @@ const taskLink = (task: Task, application: Application): string => {
   return task.title
 }
 
-export { getTaskStatus, taskLink, getCompleteSectionCount }
+const getResponses = (application: Application): ApplicationResponse => {
```

Comment:
Yeah, definitely. There's a lot of candidates in other services that could probably be taken out. It's a bit of a paradigm shift to think in a more functional way, rather than class-based all the time.